### DISCUSSION TITLE
Typos

### DISCUSSION THREAD

**Discussion by Theskyspace** [lukePeavey/quotable#80] [Quotable Data]
There are a considerable amount of typos in the quotes.

**Comment by lukePeavey** [comment]
I just checked all of the quotes for spelling/grammar using Microsoft's spell check API.  It found roughly 20 mistakes that needed to fixed. Mostly missing apostrophes in contractions (ie "weve" instead of "we've").  Is that the kind of thing you are talking about?

**Reply by Theskyspace** [reply]
Yes sir ,  and if I find any I'll inform you.
